### PR TITLE
Atomic

### PR DESCRIPTION
**Pull request by adrale**

### CODE REVIEW

**Review by ozen2 — APPROVED** [review]

**Review by axeljdev — APPROVED** [review]

**Review by JulienChevreauPro — APPROVED** [review]

File: client/src/components/atomic/checkBox/BigCheckBox.jsx
Comment (Yoann-Fortin) [inline client/src/components/atomic/checkBox/BigCheckBox.jsx:1]: It lacks the associated label tag, and too many similar ode to justify a new component.

File: client/src/components/atomic/checkBox/SmallCheckBox.jsx
Comment (Yoann-Fortin) [inline client/src/components/atomic/checkBox/SmallCheckBox.jsx:1]: It lacks the associated label tag, and too many similar ode to justify a new component.

File: client/src/components/atomic/tag/Tag.jsx
Comment (Yoann-Fortin) [inline client/src/components/atomic/tag/Tag.jsx:1]: Too many similar code to justify a new component.

File: client/src/components/atomic/tag/TagStatus.jsx
Comment (Yoann-Fortin) [inline client/src/components/atomic/tag/TagStatus.jsx:1]: It lacks the associated label tag, and too many similar ode to justify a new component.

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

File: client/package.json [hunk 1]
```diff
@@ -10,7 +10,8 @@
     "prop-types": "^15.8.1",
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
-    "react-router-dom": "^6.23.0"
+    "react-router-dom": "^6.23.0",
+    "tailwindcss-animated": "^1.1.0"
```
[the remark this answers is not in the document]
Comment (axeljdev) [reply client/package.json:14]: if i put it in dev dependencies, it doesn't work

File: client/src/components/atomic/card/CardConsultant.jsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        <Button
+          name="Téléphone"
+          img="/src/assets/images/phone.svg"
+          alt="téléphone"
+          apply="small"
+        />
+        <Button
+          name="Email"
+          img="/src/assets/images/mail.svg"
+          alt="Email"
```
Comment (KevinDavoust) [inline client/src/components/atomic/card/CardConsultant.jsx:37]: enlever les alts

**Review by KevinDavoust — CHANGES_REQUESTED** [review]

**Review by KevinDavoust — APPROVED** [review]

**Pull request merged**